functional-tests/generate-framework-agreement-signature-pages/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+. "$TESTDIR"/../setup.sh
+
+# generate-framework-agreement-signature-pages generally requires the
+# agreements repo
+[ -n "$DM_AGREEMENTS_REPO" ] || DM_AGREEMENTS_REPO="$DM_SCRIPTS_REPO/../digitalmarketplace-agreements"
+
```

Comment:
Is this file run automatically by cram?

Reply:
No the test script that wants to use it can source it. See for instance `functional-tests/generate-framework-agreement-signature-pages/signature-pages.t`.